pages/validator/external-chains/polygon-zkevm.mdx

```diff
@@ -0,0 +1,256 @@
+# Polygon ZkEVM
+
+import Markdown from 'markdown-to-jsx'
+import Tabs from '../../../components/tabs'
+import CodeBlock from '../../../components/code-block'
+import {Tabs as NextraTabs, Tab as NextraTab, Callout} from 'nextra-theme-docs'
+
+Instructions to set up your Polygon ZkEVM node.
+
+<Callout type="info" emoji="ℹ️">
+    Note: Currently ARM-based CPUs are not supported
+</Callout>
+
+## Requirements
+- [Setup your Axelar validator](/validator/setup)
+- Minimum hardware requirements: 4 cores, 32G RAM, 128G Disk with high IOPS.
+-  Ubuntu 22.04, Docker(tested on 22.04)
+- [Official Documentation](https://github.com/0xPolygonHermez/zkevm-node/blob/develop/docs/production-setup.md)
+- [Official Documentation](https://zkevm.polygon.technology/docs/setup-production-node)
+
+## Prerequisites
+This is the most straightforward path to run a zkEVM node, and it's perfectly fine for most use cases, This is quite opinionated, feel free to run this software in a different way, for instance it's not needed to use Docker, you could use the Go and C++ binaries directly.
+Some instructions to install Prerequisites are give here however be sure to check the [official documentation](https://docs.docker.com/compose/install/) to install docker compose in case this doesn't work
+```bash
+sudo apt-get install jq -y
+sudo curl -L "https://github.com/docker/compose/releases/latest/download/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+sudo chmod +x /usr/local/bin/docker-compose
+docker-compose --version
+```
+
+## Create necessary directories
+You can modify the paths as per your requirements
+
+```bash
+# ZKEVM CONFIG DIR
+mkdir -p ~/.zkevm/config
+# ZKEVM NODE STATEDB DATA DIR
+mkdir -p ~/.zkevm/data/statedb
+# ZKEVM NODE POOLDB DATA_ DIR
+mkdir -p ~/.zkevm/data/pooldb
+```
+# Get network configurations
+Storing directory paths to variables to reuse
+
+<Tabs tabs={[
+    {
+        title: "Mainnet",
+        content: <CodeBlock language="bash">
+            {`ZKEVM_NET=mainnet`}
+        </CodeBlock>
+    },
+    {
+        title: "Testnet",
+        content: <CodeBlock language="bash">
+            {`ZKEVM_NET=testnet`}
+        </CodeBlock>
+    }
+]}/>
+
+```bash
+ZKEVM_DIR=~/.zkevm
+ZKEVM_CONFIG_DIR=~/.zkevm/config
+```
+Fetch network configuration files
+```bash
+# get zip and unzip into $ZKEVM_DIR
+curl -L https://github.com/0xPolygonHermez/zkevm-node/releases/latest/download/$ZKEVM_NET.zip > $ZKEVM_NET.zip && unzip -o $ZKEVM_NET.zip -d $ZKEVM_DIR && rm $ZKEVM_NET.zip
+# create .env file from example.env
+cp $ZKEVM_DIR/$ZKEVM_NET/example.env $ZKEVM_CONFIG_DIR/.env
+```
+
+## Set node config
+Define variable to set
+```bash
+# your network could be either mainnet or tetsnet, we defined this earlier
+ZKEVM_NETWORK=$ZKEVM_NET
+# L1 node rpc
+ZKEVM_NODE_ETHERMAN_URL="your_ETH_rpc"
```

Comment:
add a note that this must point to the validator's ethereum rpc node for security

Reply:
Added